modin/pandas/test/test_io.py

```diff
@@ -297,6 +298,32 @@ def make_csv_file():
                 pass
 
 
+@pytest.fixture(scope="class")
+def TestReadCSVFixture():
+    """TestReadCSVFixture makes temp csv files that
+    can be used by several test functions.
+    """
+    filenames = []
+    files_ids = [
+        "test_read_csv_regular",
+    ]
+    pytest.csvs_names = {file_id: get_unique_filename(file_id) for file_id in files_ids}
```

Comment:
Why are you modificate external package?

Reply:
This is the way `pytest` implements global variables.